airflow/api/auth/backend/auth0_auth.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this general enough to contribute back to Airflow?

Reply:
Yes, for basic usage. But I guess some people may have some additional usages like maintain user profile sent back with auth0 response.